hail/python/hailtop/hail_logging.py

```diff
@@ -15,7 +16,7 @@ def add_fields(self, log_record, record, message_dict):
 def configure_logging():
     fmt = CustomJsonFormatter('(levelname) (asctime) (filename) (funcNameAndLine) (message)')
 
-    stream_handler = logging.StreamHandler()
+    stream_handler = logging.StreamHandler(stream=sys.stdout)
```

Comment:
will this cause all log messages to go to stdout including errors? Also, how do we make sure we can still see the logs with kubectl logs?

Reply:
Yes, all log messages will go to stdout. Kubectl shows stdout and stderr so all our logs will still be visible in kubectl logs.